### PR TITLE
use correct attribute

### PR DESCRIPTION
**Pull request by snopoke**
http://manage.dimagi.com/default.asp?185229
@gcapalbo

### CODE REVIEW

**Comment by czue** [comment]
:+1: will let @gcapalbo take a look

File: corehq/apps/hqmedia/models.py [hunk 1]
```diff
@@ -402,7 +402,7 @@ class HQMediaMapItem(DocumentSchema):
 
     @property
     def url(self):
-        return reverse("hqmedia_download", args=[self.media_type, self.media_id]) if self.media_id else ""
+        return reverse("hqmedia_download", args=[self.media_type, self.multimedia_id]) if self.media_id else ""
```
Comment (gcapalbo) [inline corehq/apps/hqmedia/models.py]: does the other reference to `media_id` need to be updated too?

File: corehq/apps/hqmedia/models.py [hunk 2]
```diff
@@ -402,7 +402,7 @@ class HQMediaMapItem(DocumentSchema):
 
     @property
     def url(self):
-        return reverse("hqmedia_download", args=[self.media_type, self.media_id]) if self.media_id else ""
+        return reverse("hqmedia_download", args=[self.media_type, self.multimedia_id]) if self.multimedia_id else ""
```
Comment (dimagimon) [inline corehq/apps/hqmedia/models.py]: E501 line too long (116 > 115 characters)

File: corehq/apps/hqmedia/models.py (hunk 1 above)
Comment (snopoke) [inline corehq/apps/hqmedia/models.py]: updated

On 23 October 2015 at 13:45, gcapalbo <notifications@github.com> wrote:

> In corehq/apps/hqmedia/models.py
> <https://github.com/dimagi/commcare-hq/pull/8791#discussion_r42856290>:
>
> > @@ -402,7 +402,7 @@ class HQMediaMapItem(DocumentSchema):
> >
> >      @property
> >      def url(self):
> > -        return reverse("hqmedia_download", args=[self.media_type, self.media_id]) if self.media_id else ""
> > +        return reverse("hqmedia_download", args=[self.media_type, self.multimedia_id]) if self.media_id else ""
>
> does the other reference to media_id need to be updated too?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/8791/files#r42856290>.
>



-- 
Simon Kelly
Senior Engineer | Dimagi South Africa

**Pull request merged**